connectors/github-pr/src/main/java/com/vmware/connectors/github/pr/v3/Review.java

```diff
@@ -0,0 +1,72 @@
+package com.vmware.connectors.github.pr.v3;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+/**
+ * https://developer.github.com/v3/pulls/reviews/
+ */
+@AutoProperty
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Without this, github isn't happy about an explicit null for the body property when sending sending in an approve request.